### PR TITLE
JSON parse voodoo ids

### PR DESCRIPTION
**Pull request by heidar**

### CODE REVIEW

File: spec/cassettes/voodoo_sms/send.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ http_interactions:
       - application/json
     body:
       encoding: UTF-8
-      string: '{"result":200,"resultText":"200 OK","reference_id": "4103395"}'
+      string: '{"result":200,"resultText":"200 OK","reference_id": ["4103395"]}'
```
Comment (heidar) [inline spec/cassettes/voodoo_sms/send.yml]: Can someone double check this is actually matches a real response?

Comment (heidar) [reply spec/cassettes/voodoo_sms/send.yml]: Double checked, it's exactly like that. 👍

**Comment by AdamWhittingham** [comment]
This is looking good- just want to be sure what happens to in-flight messages. Do we need to migrate all the current references in the DB to be JSON decoded?

**Comment by heidar** [comment]
@AdamWhittingham that's done in the branch that implements this upgrade.

**Comment by AdamWhittingham** [comment]
Well that makes perfect sense then!

**Pull request merged**